index.html

```diff
@@ -2489,6 +2489,16 @@ <h3 property="schema:name">Social Web Working Group</h3>
         </p>
       </section>
 
+      <section id="security-recursive-objects">
+        <h2>Recursive Objects</h2>
+        <p>
+	  Servers should take care to properly handle recursive ActivityStreams
```

Comment:
@jaywink perhaps something like:

> Servers should set a limit on how deep to recurse while resolving objects, or otherwise specially handle ActivityStreams objects with recursive references.

Does that seem better to you? It's a bit wordy but I feel like we shouldn't imply that setting a recursion limit is the only way to handle this; there are probably more intelligent things you can do